product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx

```diff
@@ -595,3 +595,11 @@ bdr.alter_node_option(node_name text,
 -   `node_name` &mdash; Name of the node to be changed.
 -   `config_key` &mdash; Key of the option in the node to be changed.
 -   `config_value` &mdash; New value to be set for the given key.
+
+The node options that can be changed using this function are:
+- `route_priority` &mdash; Relative routing priority of the node against other nodes in the same node group. Default is '100'.
```

Comment:
I think default is `-1`. TPA is setting it to 100 for data nodes in TPA created cluster using config.yml but default is not that.